README.md

```diff
@@ -0,0 +1,41 @@
+# Encryptonize&reg; library
+[![Library build](https://github.com/cyber-crypt-com/encryptonize-lib/actions/workflows/build.yaml/badge.svg)](https://github.com/cyber-crypt-com/encryptonize-lib/actions/workflows/build.yaml)
+[![Go Report Card](https://goreportcard.com/badge/github.com/cyber-crypt-com/encryptonize-lib)](https://goreportcard.com/report/github.com/cyber-crypt-com/encryptonize-lib)
+[![Go Reference](https://pkg.go.dev/badge/github.com/cyber-crypt-com/encryptonize-lib)](https://pkg.go.dev/github.com/cyber-crypt-com/encryptonize-lib)
+
+The Encryptonize&reg; library provides cryptographic functions for data encryption and decryption, as well as enforcing access control around the encrypted data. It allows developers to easily implement various security schemes and protocols for protecting data.
+
+## Features
+- **Authenticated Encryption with Associated Data (AEAD)**
+    - binary data can be encrypted/decrypted using the standardized AES256-GCM algorithm
+    - the Ciphertext and Associated Data can be securely stored in untrusted locations as the recovery of the plaintext is computationally infeasible without the Encryption Key, and any changes to the Ciphertext or Associated Data will be detected during the decryption process
+- **Key Wrapping with Padding Authenticated Encryption (KWP-AE)**
+    - each object is encrypted with a random 256-bit key which is then wrapped with the configured Wrapping Key using the standardized KWP-AE algorithm
+    - wrapped keys can be securely stored in untrusted locations (alongside the encrypted objects) as the recovery of the plaintext key is computationally infeasible without the Wrapping Key, and any changes to the Wrapped Key will be detected during the unwrapping process
+- **Discretionary Access Control**
+    - each encrypted object has individual access controls defined by the data owner
+    - data owners decide which users or user groups can access the decrypted objects
+- **User Authentication**
+    - authentication is done with custom credentials generated by the library functions
+- **Searcheable Symmetric Encryption (SSE)**
```

Comment:
We should probably mention what the security gains are for this feature.

Reply:
Fixed